content/influxdb/cloud-serverless/process-data/downsample_quix.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Downsample data stored in InfluxDB using Quix Streams
+description: >
+  Query time series data stored in InfluxDB and regular intervals, write it to Kafka and continuously downsample it using Quix Streams, then write the
+  downsampled data back to InfluxDB.
+menu:
+  influxdb_cloud_serverless:
+    name: Downsample data with Quix
+    parent: Process & visualize data
+weight: 101
+related:
+  - /influxdb/cloud-serverless/query-data/sql/aggregate-select/, Aggregate or apply selector functions to data (SQL)
+---
+
+Query time series data stored in InfluxDB and regular intervals, write it to Kafka and continuously downsample it using Quix Streams, then write the
+  downsampled data back to InfluxDB.
+
+This guide uses [Python](https://www.python.org/) and the
+[InfluxDB v3 Python client library](https://github.com/InfluxCommunity/influxdb3-python),
+but you can use your runtime of choice and any of the available
+[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/).
+
+The downsampling process uses [Quix Streams](https://github.com/quixio/quix-streams). Quix Streams is an open source Python library for 
+building containerized stream processing applications with Apache Kafka.
+It is designed to run as a service that continuously processes a stream of data while streaming the results to a Kafka topic. You can try it locally, with a local
+Kafka installation, or  run it in [Quix Cloud](https://quix.io/) for a free trial.
+
+## Pipeline architecture
+
+The following diagram illustrates how data is passed between processes as it is downsampled:
+
+![downsampling pipeline architecture](quix-pipeline-downsample.png)
+
+Note that it is usually more efficient to write your raw data directly to Kafka rather than writing raw data to InfluxDB first (i.e. start the pipeline with the “raw-data” topic). However, this guide assumes that you already have raw data in an InfluxDB instance that you would like to downsample.
```

Comment:
```suggestion
{{% note %}}
It is usually more efficient to write raw data directly to Kafka rather than
writing raw data to InfluxDB first (essentially starting the Quix Streams
pipeline with the "raw-data" topic). However, this guide assumes that you
already have raw data in InfluxDB that you want to downsample.
{{% /note %}}
```